.github/workflows/build.yml

```diff
@@ -47,3 +47,56 @@ jobs:
           artifactory-reader-role: private-reader    # Override default public-reader
           artifactory-deployer-role: qa-deployer     # Override default public-deployer
           gradle-args: "-Dsonar.sca.exclusions=integrationTests/src/test/resources/**,src/test/projects/**"
+
+  qa:
+    strategy:
+      fail-fast: false
+      matrix:
+        item:
+          - gradle_version: &gradle-version-to-test-on-windows-too "9.2.0"
```

Comment:
Why is the windows task configuration referenced here?

Reply:
We have the same understanding: the latest version is the only one that should run on both platforms.

But for clarity, I think it would be best to rename to the variable to something like `latest-gradle-version`. The fact that a seemingly OS-agnostic value has an OS in its name is probably what confused me the most. WDYT?